sources/packages/web/src/components/generic/ModalDialogBase.vue

```diff
@@ -7,7 +7,12 @@
     scrollable
     :fullscreen="showFullScreen"
   >
-    <v-card elevation="10" :max-width="maxWidth" class="modal-height">
+    <v-card
+      elevation="10"
+      :max-width="showFullScreen ? undefined : maxWidth"
+      :min-width="showFullScreen ? undefined : minWidth"
```

Comment:
Just to verify, is smAndDown(showFullScreen) reactive and not required to use through computed property?

Reply:
Yes, it should be since the UI is adapting as expected and the previously existing smAndDown was assigned to the `v-dialog` fullscreen property.